domains/Discussions/Controllers/QuestionsStoreController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Domains\Discussions\Controllers;
+
+use App\Http\Controllers\Controller;
+use Domains\Discussions\Models\Question;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+
+class QuestionsStoreController extends Controller
+{
+    public function __invoke(Request $request): Response
+    {
+        $this->validate($request, [
+            'title' => ['required', 'string', 'max:255'],
+            'description' => ['required', 'string'],
+        ]);
+
+        (new Question())
```

Comment:
Let's inject this Question class as a dependency from the constructor and reference it here, like it's done in the `LinksStoreController` for example.

Reply:
done